accelerator/apis.c

```diff
@@ -290,7 +291,8 @@ status_t api_mam_send_message(const iota_config_t* const tangle, const iota_clie
 done:
   // Save and destroying MAM API
   if (ret != SC_MAM_FAILED_INIT) {
-    if (mam_api_save(&mam, tangle->mam_file) || mam_api_destroy(&mam)) {
+    if (mam_api_save(&mam, tangle->mam_file, (const tryte_t*)tangle->mam_encr_key, mam_encr_key_len) ||
```

Comment:
We don't need a encr_key, just set this to NULL. Besides, I'm going to set this deprecated. The mam API can generate certain channel if parameters are provided.